v3/src/components/map/utilities/map-utils.ts

```diff
@@ -70,7 +70,7 @@ export const boundaryAttributeFromDataSet = (dataSet: IDataSet) => {
  */
 export const getLatLongBounds = (dataConfiguration: IDataConfigurationModel) => {
   const getRationalLongitudeBounds = (longs: number[]) => {
-      longs.sort()
+      longs.sort((v1:number, v2:number) => (v1 - v2))
```

Comment:
```suggestion
      longs.sort((v1: number, v2: number) => (v1 - v2))
```
would be the more conventional formatting. I had to look it up to remind myself why this fixes the bug -- the default behavior of `sort()` is to convert the contents to strings and sort them alphabetically. 🤷